src/library/library_prefs.cpp

```diff
@@ -13,16 +13,41 @@ const ConfigKey mixxx::library::prefs::kLegacyDirectoryConfigKey =
 const QString mixxx::library::prefs::kConfigGroup =
         QStringLiteral("[Library]");
 
+const ConfigKey mixxx::library::prefs::kRescanOnStartupConfigKey =
+        ConfigKey{
+                mixxx::library::prefs::kConfigGroup,
+                QStringLiteral("RescanOnStartup")};
+
 const ConfigKey mixxx::library::prefs::kKeyNotationConfigKey =
         ConfigKey{
                 mixxx::library::prefs::kConfigGroup,
                 QStringLiteral("key_notation")};
 
+const ConfigKey mixxx::library::prefs::kTrackDoubleClickActionConfigKey =
+        ConfigKey{
+                mixxx::library::prefs::kConfigGroup,
+                QStringLiteral("TrackLoadAction")};
+
 const ConfigKey mixxx::library::prefs::kEditMetadataSelectedClickConfigKey =
         ConfigKey{
                 mixxx::library::prefs::kConfigGroup,
                 QStringLiteral("EditMetadataSelectedClick")};
 
+const ConfigKey mixxx::library::prefs::kHistoryCleanupMinTracksConfigKey =
+        ConfigKey{
+                mixxx::library::prefs::kConfigGroup,
+                QStringLiteral("history_cleanup_min_tracks")};
+
+const ConfigKey mixxx::library::prefs::kHistoryCleanupKeepLockedConfigKey =
+        ConfigKey{
+                mixxx::library::prefs::kConfigGroup,
+                QStringLiteral("history_cleanup_keep_locked")};
+
+const ConfigKey mixxx::library::prefs::kHistoryRecentlyPlayedThresholdConfigKey =
+        ConfigKey{
+                mixxx::library::prefs::kConfigGroup,
+                QStringLiteral("history_recently_played_threshold")};
```

Comment:
`history_track_readd_threshold`?
If you don't consider that suitable either I'd rather add a comment here than creating a foot-long configkey.

In the preferences the spinbox tooltip explains the meaning. Do you think that's not sufficient?

Reply:
what about
'Recently played' threshold to prevent duplicate entries
?

Reply:
"history_track_readd_threshold" works better for me, while "threshold" can be anything. That's why I had proposed "gap" or "distance"

> 'Recently played' threshold to prevent duplicate entries

This can be improved because it will not "prevent duplicates" it just sets the minimal distance of duplicates. 

"Track re-add threshold" works also in the GUI 

but how about: 
"Minimum distance of repeated tracks"? 

and for the config key 

"history_track_repeat_distance"

Reply:
"history_duplicate_distance"

Reply:
Thanks, I wasn't really in the shape to come up with a proper term.
I'll go with `history_track_duplicate_distance` and for the History preference "Track duplicate distance"